### DISCUSSION TITLE
The red number next to a room tile should be the number of notifications, not the number of unreads

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#1985] [Product Feedback]
This is an internal discussion we've been having. For internal folks the link is here: https://github.com/vector-im/wat-internal/discussions/10

For externals: 

> Currently, when you have a single notification (e.g. a mention) in a room which has 100 unread messages, we display a big red 100 next to the room name.

> To me, from my experience using other web sites and apps, that means "you have 100 things to do".

> But in reality, you only have "red" 1 thing to do, and 100 "grey" things to do.

> We should either:

> * Not display 100 at all, but instead display "1", or the number of notifications. The 100 part is less important than the 1; or
> * Display both the 1 and the 100, in different colours.

**Comment by daniellekirkwood** [comment]
@andybalaam created the discussion here and linked to our internal one just so that when we look thru the external discussions for prioritisation we'll see it represented. Also, some community members may have thoughts...

**Comment by MarianRychtecky** [comment]
The way I process is that I need to read 100 to get a full context, not just one. Having only the number of mentions is confusing for me.
